### ISSUE TITLE
resolve signTransaction

### ISSUE THREAD

**Issue by Pecking** [steemit/steem-js#403]
I wanted to sign my own data structure through the signedTransaction method, and then I called the verifyAuthority method to verify that my signature data returned "err Error: missing required active authority: Missing Active Authority initminer" and I made sure my activity was correct.

**Comment by Pecking** [comment]
I wanted to sign my own data structure through the signedTransaction method, and then I called the verifyAuthority method to verify that my signature data returned "err Error: missing required active authority: Missing Active Authority initminer" and I made sure my activity was correct.

**Comment by roadscape** [comment]
Could you rephrase in the form of a question, explain what you expect to happen, and/or share a code snippet with which one could reproduce the issue?

**Comment by Pecking** [comment]
describe:I need to encrypt a custom data structure and send it to our server ,Of course, our server added this method.

normal data：    
![12](https://user-images.githubusercontent.com/27414779/45497751-1be3ba80-b7ab-11e8-9667-11147da0b163.jpg)
             
my normal  data：
![cache_-18d39178d2795e90](https://user-images.githubusercontent.com/27414779/45497703-04a4cd00-b7ab-11e8-9ea4-1562c00a1f6e.jpg)

result: Our service returns my signature error(Error: missing required active authority: Missing Active Authority initminer" )
![56](https://user-images.githubusercontent.com/27414779/45497822-459ce180-b7ab-11e8-94b2-879330fc811e.jpg)
*I guess this mistake should be a mistake in the signature calculation, but I don't know how to solve this problem.

**Comment by roadscape** [comment]
It looks like you are working with a STEEM fork; I don't think it's an issue with steem-js as it's known to support other chains, so you need to double-check config. Also these topics may help:
https://github.com/steemit/steem/issues/2453
https://github.com/steemit/steem/issues/2683
